cmd/keeper/cmd/keeper.go

```diff
@@ -1812,25 +1826,35 @@ func keeper(c *cobra.Command, args []string) {
 		die("cannot create data dir: %v", err)
 	}
 
-	if cfg.pgListenAddress == "" {
-		die("--pg-listen-address is required")
+	if cfg.pgAdvertiseAddress == "" && cfg.pgListenAddress == "" {
+		die("--pg-listen-address and/or --pg-advertise-address is required")
+	} else if cfg.pgListenAddress == "" {
+		cfg.pgListenAddress = "*"
```

Comment:
I'll prefer the user to always specifiy the listen address and not default to '*' when not specified.

Reply:
oki, no problem, I'm quizzing myself for a while about the default values to apply.